doc/manual/src/language/import-from-derivation.md

```diff
@@ -0,0 +1,139 @@
+# Import From Derivation
+
+The value of a Nix expression can depend on the contents of a [store object].
+In this case, when that store object is needed, evaluation will be paused, the store object [realised], and then evaluation resumed.
+
+[store object]: @docroot@/glossary.md#gloss-store-object
+[derivation]: @docroot@/glossary.md#gloss-derivation
+[realised]: @docroot@/glossary.md#gloss-realise
+
+This has performance implications:
+Since evaluation is sequential, each required store object that is not already in the store will also be realised sequentially.
+Usually, if store objects are not already present, realisation is orders of magnitude slower than evaluation.
```

Comment:
Not sure I quite got it but 
```suggestion
Since Nix's evaluator is sequential, Nix only discovers the store objects a Nix program requires one at a time.
If a given Nix program requires many store objects, and they are not already present realizing them one at a time is much slow than realizing them at once.  
```

1. "Orders of magnitude" means something precise, but without being concrete about what we are comparing we cannot be precise.
2. Realization itself still is parallel, just figuring out what to realize is sequential. (I still don't think my version fixes this enough.) It is an important distinction cause it means one can "batch" all the IFD at once if one is careful, which makes things a lot faster! E.g. I did that in https://github.com/obsidiansystems/obelisk/pull/1040

Obviously covering this technique is out of scope for the reference, but we should still be precise enough so that when `nix.dev` does the tutorial, the reference doesn't contradict it.

Even my "one at a time" is also wrong. Nix realizes string contexts one at a time, but a string context is a set which can refer to arbitrary many store objects. (And even if that weren't true, one could create a derivation whose only purpose is to create a store object referring to many other store objects, achieving the same outcome.)